tests/testsuite/conda.rs

```diff
@@ -0,0 +1,29 @@
+use ansi_term::Color;
+use std::io;
+
+use crate::common;
+
+#[test]
+fn not_in_env() -> io::Result<()> {
+    let output = common::render_module("conda").env_clear().output()?;
+
+    let expected = "";
+    let actual = String::from_utf8(output.stdout).unwrap();
+
+    assert_eq!(expected, actual);
+    Ok(())
+}
+
+#[test]
+fn env_set() -> io::Result<()> {
+    let output = common::render_module("conda")
+        .env_clear()
```

Comment:
Just FYI, [Render module already does this for us](https://github.com/starship/starship/blob/master/tests/testsuite/common.rs#L32).

Reply:
I just copied these from the [aws testsuite](https://github.com/starship/starship/blob/master/tests/testsuite/aws.rs), basically

Reply:
Alright, looks like we'll have some cleanup work to do in the future. Thanks for letting me know.